packages/gatsby-docs/src/pages/run/running_the_tool.mdx

```diff
@@ -26,8 +26,9 @@ Warning: Remember that the temporary output folder is wiped when the UI Upgrade
 
 ![1. Press F1, 2. Type "Run SPM UI Upgrade Helper", 3. Select the shortcut, 4. Wait for the tool to finish, 5. Click "Source Control: Git" to see the output](../../images/upgrade-helper.gif "Running the UI Upgrade Helper tool")
 
-8. Copy the contents of `<output folder>` into your v8 development environment.
-9. Build and test v8.
+8. For more information about Window Size Tool changes, see [Reviewing Window Size Tool changes](https://github.com/IBM/spm-ui-upgrade-helper/blob/poraghal-docs-review-WST/packages/gatsby-docs/src/pages/setup/tools.mdx#reviewing-window-size-tool-changes).
```

Comment:
The url in this link needs to be changed to a relative one so it points to the right place in production. e.g.

```suggestion
8. For more information about Window Size Tool changes, see [Reviewing Window Size Tool changes](../../setup/tools#reviewing-window-size-tool-changes).
```